portfolio/src/main/webapp/script.js

```diff
@@ -64,3 +64,239 @@ function clearComments() {
 function deleteComments() {
   fetch('/delete-data', {method: 'POST'}).then(() => getComments());
 }
+
+function createMap() {
+  // Create a map with a dark color scheme centered at my hometown, Anaheim
+  const map = new google.maps.Map(
+      document.getElementById('map'), {
+        center: {lat: 33.835982, lng: -117.914157},
+        zoom: 11,
+        styles: [
+          {elementType: 'geometry', stylers: [{color: '#292929'}]},
```

Comment:
I think it'd be nice to be consistent by having line breaks.

ex:
{
  elementType: ...,
  stylers: ...
}

Reply:
Done.